.github/workflows/rust.yml

```diff
@@ -22,32 +22,42 @@ jobs:
       - name: Check
         run: cargo check --verbose --workspace --all-targets
 
-  # Library tests (run in parallel with "tests" and dpcs)
-  test-lib:
+  # Tests for tpchgen
```

Comment:
I tried to parallelize the tests more by running the tpchgen tests in their own job in parallel to the tests for tpchgen-arrow